Dockerfile

```diff
@@ -1,16 +1,14 @@
 ############################
 # STEP 1 build executable binary
 ############################
-FROM golang:alpine AS builder
-# Install git.
-# Git is required for fetching the dependencies.
-RUN apk update && apk add --no-cache git
```

Comment:
Is git no longer required?

Reply:
It's still required, but it's included in the go-toolset image where it wasn't in the minimal golang:alpine image we were using previously.